Dockerfile

```diff
@@ -18,12 +18,7 @@
 # granted to it by virtue of its status as an Intergovernmental Organization or
 # submit itself to any jurisdiction.
 
-FROM ubuntu:16.04
-RUN apt-get update
-RUN apt-get -y upgrade
-RUN apt-get -y install rabbitmq-server
-RUN apt-get -y autoremove && apt-get -y clean
-RUN service rabbitmq-server start
+FROM rabbitmq:3.6-alpine
 ADD start.sh /start.sh
```

Comment:
Indeed, I have just tested and it works. The change requires to switch the connections [here](https://github.com/reanahub/reana-workflow-controller/blob/master/reana_workflow_controller/tasks.py#L35) for `reana-workflow-controller` and [here](https://github.com/reanahub/reana-workflow-engine-yadage/blob/master/reana_workflow_engine_yadage/celeryapp.py#L27) for `reana-workflow-engine-yadage` to `guest:guest`. However, since I had to rebuild the images to test it, the 'extracting hardcoded strings' sprint becomes more important in order to stop having this problems.